app/src/main/java/com/android/sample/ui/calendar/CalendarScreen.kt

```diff
@@ -196,68 +143,114 @@ fun DaySection(
     navigationActions: NavigationActions,
     calendarViewModel: CalendarViewModel
 ) {
+  val context = LocalContext.current
   Row(
       modifier =
           Modifier.fillMaxWidth()
               .testTag("daySection")
               .padding(vertical = 8.dp)
-              .background(
-                  color = MediumGrey.copy(alpha = 0.3f), shape = MaterialTheme.shapes.medium)
+              .background(color = Color.Transparent, shape = MaterialTheme.shapes.medium)
               .padding(16.dp)
               .clickable {
                 calendarViewModel.updateSelectedDate(date)
                 navigationActions.navigateTo(Screen.DAY_CALENDAR)
               },
-      horizontalArrangement = Arrangement.SpaceBetween,
       verticalAlignment = Alignment.CenterVertically) {
-        Column(modifier = Modifier, horizontalAlignment = Alignment.Start) {
-          Text(
-              text = date.dayOfMonth.toString(),
-              style = MaterialTheme.typography.headlineLarge,
-              fontWeight = FontWeight.Bold)
-          Spacer(modifier = Modifier.height(4.dp))
-          Text(
-              text = getMonthName(date.monthNumber),
-              style = MaterialTheme.typography.bodySmall,
-              fontWeight = FontWeight.Light,
-              color = DarkGrey)
-        }
-
+        Column(
+            modifier = Modifier,
+            horizontalAlignment = Alignment.Start,
+            verticalArrangement = Arrangement.Center) {
+              Spacer(modifier = Modifier.height(40.dp))
+              Box(
+                  modifier =
+                      Modifier.drawBehind {
+                        drawIntoCanvas { canvas ->
+                          shadowText(
+                              canvas, context, String.format("%02d", date.dayOfMonth), 50.sp.toPx())
+                        }
+                      })
+              Spacer(modifier = Modifier.height(20.dp))
+              Box(
+                  modifier =
+                      Modifier.drawBehind {
+                        drawIntoCanvas { canvas ->
+                          shadowText(canvas, context, getMonthName(date.monthNumber), 16.sp.toPx())
+                        }
+                      })
+            }
+        Spacer(modifier = Modifier.width(150.dp))
         if (workouts.isEmpty()) {
           Text(
-              text = "No workout", style = MaterialTheme.typography.bodyMedium, color = NeutralGrey)
+              text = stringResource(id = R.string.NoWorkout),
```

Comment:
Good idea to change this into a stringResource instead of directly working with strings